### PR TITLE
don't translate brand colors

### PR DESCRIPTION
**Pull request by btkostner**
Fixes translatable brand colors

### Changes Summary

- l10n ignore brand colors
- easier to understand translations checker

### CODE REVIEW

**Comment by lewisgoddard** [comment]
ok to check

**Comment by lewisgoddard** [comment]
ok to test

File: backend/translations-checker.php [hunk 1]
```diff
@@ -43,5 +43,12 @@ function globRecursive($Pattern, $Flags = 0) {
...
-// echo json_encode($result);
+$invalidCount = count($result['invalid_files']);
+
+echo count($result['valid_files'])." valid translation files\n";
+
+if ($invalidCount) {
+    var_dump($result['invalid_files']);
+}
+
+echo count($result['invalid_files'])." invalid translation files\n";
```
Comment (lewisgoddard) [inline backend/translations-checker.php]: why not `$invalidCount` ?

Comment (btkostner) [inline backend/translations-checker.php]: forgot to change. fixed

**Pull request merged**